assets/queries/cloudFormation/iam_user_passwordResetRequired_false/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "iam_user_missing_PasswordResetRequired",
+  "queryName": "user IAM missing PasswordResetRequired",
+  "severity": "MEDIUM",
+  "category": null,
```

Comment:
Missing category

Reply:
Yes. These queries were all without category defined. However, according to previous ones, I assume the best one for this is "Identity and Access Management"

Reply:
Yes, that is right!